diff -Naur mock-1.4.13/py/mockbuild/plugins/root_cache.py mock-1.4.13.tpg/py/mockbuild/plugins/root_cache.py
--- mock-1.4.13/py/mockbuild/plugins/root_cache.py	2018-10-27 03:07:21.914735000 +0000
+++ mock-1.4.13.tpg/py/mockbuild/plugins/root_cache.py	2018-10-27 03:11:55.374731428 +0000
@@ -157,11 +157,11 @@
         for m in self.buildroot.mounts.get_mountpoints():
             if m.startswith('/'):
                 if m.startswith(br_path):
-                    self.exclude_tar_cmds.append('--exclude=.%s' % m[len(br_path):])
+                    self.exclude_tar_cmds.append('--exclude=^%s' % m[len(br_path):])
                 else:
-                    self.exclude_tar_cmds.append('--exclude=.%s' % m)
+                    self.exclude_tar_cmds.append('--exclude=^%s' % m)
             else:
-                self.exclude_tar_cmds.append('--exclude=./%s' % m)
+                self.exclude_tar_cmds.append('--exclude=^./%s' % m)
 
     @traceLog()
     def _rootCachePostInitHook(self):
diff -Naur mock-1.4.13/py/mockbuild/util.py mock-1.4.13.tpg/py/mockbuild/util.py
--- mock-1.4.13/py/mockbuild/util.py	2018-10-27 03:07:21.911735000 +0000
+++ mock-1.4.13.tpg/py/mockbuild/util.py	2018-10-27 03:10:20.715732737 +0000
@@ -874,7 +874,7 @@
             'dir': "%(cache_topdir)s/%(root)s/root_cache/",
             'tar': "gnutar",
             'compress_program': 'pigz',
-            'exclude_dirs': ["./proc", "./sys", "./dev", "./tmp/ccache", "./var/cache/yum", "./var/cache/dnf"],
+            'exclude_dirs': ["^./proc", "^./sys", "^./dev", "^./tmp/ccache", "^./var/cache/yum", "^./var/cache/dnf"],
             'extension': '.gz'},
         'bind_mount_enable': True,
         'bind_mount_opts': {
