diff --git a/py/mockbuild/util.py b/py/mockbuild/util.py
index 3e00f80..12e299a 100644
--- a/py/mockbuild/util.py
+++ b/py/mockbuild/util.py
@@ -930,7 +930,7 @@ def setup_default_config_opts(unprivUid, version, pkgpythondir):
             'tar': "gnutar",
             'compress_program': 'pigz',
             'decompress_program': None,
-            'exclude_dirs': ["./proc", "./sys", "./dev", "./tmp/ccache", "./var/cache/yum", "./var/cache/dnf"],
+            'exclude_dirs': [],
             'extension': '.gz'},
         'bind_mount_enable': True,
         'bind_mount_opts': {
