diff -up mock-2.16/docs/site-defaults.cfg.omv~ mock-2.16/docs/site-defaults.cfg
--- mock-2.16/docs/site-defaults.cfg.omv~	2022-06-18 21:01:16.849002114 +0200
+++ mock-2.16/docs/site-defaults.cfg	2022-06-18 21:01:25.223072120 +0200
@@ -119,7 +119,7 @@
 # config_opts['yum_command'] = '/usr/bin/yum'
 # config_opts['yum_builddep_command'] = '/usr/bin/yum-builddep'
 # config_opts['dnf_command'] = '/usr/bin/dnf'
-# config_opts['rpm_command'] = '/bin/rpm'
+# config_opts['rpm_command'] = '/usr/bin/rpm'
 # config_opts['rpmbuild_command'] = '/usr/bin/rpmbuild'
 #
 # By default a Yum/DNF update is performed before each rebuild
diff -up mock-2.16/py/mockbuild/config.py.omv~ mock-2.16/py/mockbuild/config.py
--- mock-2.16/py/mockbuild/config.py.omv~	2022-06-18 21:00:57.053836116 +0200
+++ mock-2.16/py/mockbuild/config.py	2022-06-18 21:01:05.632908148 +0200
@@ -305,7 +305,7 @@ def setup_default_config_opts(unprivUid,
     config_opts['microdnf_builddep_command'] = '/usr/bin/dnf'
     config_opts['microdnf_builddep_opts'] = []
     config_opts['microdnf_common_opts'] = []
-    config_opts['rpm_command'] = '/bin/rpm'
+    config_opts['rpm_command'] = '/usr/bin/rpm'
     config_opts['rpmbuild_command'] = '/usr/bin/rpmbuild'
     config_opts['dnf_disable_plugins'] = ['local', 'spacewalk', 'versionlock']
     config_opts['user_agent'] = "Mock ({{ root }}; {{ target_arch }})"
