diff -up mock-1.4.14/py/mockbuild/plugins/root_cache.py.omv~ mock-1.4.14/py/mockbuild/plugins/root_cache.py
--- mock-1.4.14/py/mockbuild/plugins/root_cache.py.omv~	2019-04-30 02:31:38.177422968 +0200
+++ mock-1.4.14/py/mockbuild/plugins/root_cache.py	2019-04-30 02:38:41.059371592 +0200
@@ -58,7 +58,11 @@ class RootCache(object):
         plugins.add_hook("postchroot", self._rootCachePostShellHook)
         plugins.add_hook("postyum", self._rootCachePostShellHook)
         self.exclude_dirs = self.root_cache_opts['exclude_dirs']
-        self.exclude_tar_cmds = ["--exclude=" + item for item in self.exclude_dirs]
+        if self.config['tar'] == 'bsdtar':
+            anchor = '^'
+        else:
+            anchor = ''
+        self.exclude_tar_cmds = ["--exclude=" + anchor + item for item in self.exclude_dirs]
 
     # =============
     # 'Private' API
diff -up mock-1.4.14/py/mockbuild/util.py.omv~ mock-1.4.14/py/mockbuild/util.py
