diff -up mock-1.4.9/py/mockbuild/util.py.omv~ mock-1.4.9/py/mockbuild/util.py
--- mock-1.4.9/py/mockbuild/util.py.omv~	2018-05-06 22:17:39.501612694 +0200
+++ mock-1.4.9/py/mockbuild/util.py	2018-05-06 22:19:22.693382823 +0200
@@ -69,6 +69,14 @@ PER_LINUX = 0x0000
 personality_defs = {
     'x86_64': PER_LINUX, 'ppc64': PER_LINUX, 'sparc64': PER_LINUX,
     'i386': PER_LINUX32, 'i586': PER_LINUX32, 'i686': PER_LINUX32,
+    'armv7': PER_LINUX32, 'armv7l': PER_LINUX32, 'armv7hl': PER_LINUX32,
+    'armv7hnl': PER_LINUX32, 'armv7hcnl': PER_LINUX32,
+    'armv7b': PER_LINUX32, 'armv7hb': PER_LINUX32,
+    'armv7hnb': PER_LINUX32, 'armv7hcnb': PER_LINUX32,
+    'armv8': PER_LINUX32, 'armv8l': PER_LINUX32, 'armv8hl': PER_LINUX32,
+    'armv8hnl': PER_LINUX32, 'armv8hcnl': PER_LINUX32,
+    'armv8b': PER_LINUX32, 'armv8hb': PER_LINUX32,
+    'armv8hnb': PER_LINUX32, 'armv8hcnb': PER_LINUX32,
     'ppc': PER_LINUX32, 'sparc': PER_LINUX32, 'sparcv9': PER_LINUX32,
     'ia64': PER_LINUX, 'alpha': PER_LINUX,
     's390': PER_LINUX32, 's390x': PER_LINUX,
