diff -up mock-6.7/py/mockbuild/backend.py.4~ mock-6.7/py/mockbuild/backend.py
--- mock-6.7/py/mockbuild/backend.py.4~	2026-03-03 12:39:58.000000000 +0100
+++ mock-6.7/py/mockbuild/backend.py	2026-03-06 21:36:27.076141714 +0100
@@ -60,7 +60,7 @@ class Commands(object):
         self.no_root_shells = config['no_root_shells']
 
         self.private_network = not config['rpmbuild_networking']
-        self.rpmbuild_noclean_option = None
+        self.rpmbuild_noclean_option = "--noclean"
 
         # on-demand buildroot properties
         # spec path in buildroot
