--- mock-1.4.9/py/mockbuild/plugins/root_cache.py.omv~	2018-03-06 21:17:16.181113875 +0100
+++ mock-1.4.9/py/mockbuild/plugins/root_cache.py	2018-03-06 21:23:37.616930340 +0100
@@ -39,7 +39,6 @@ class RootCache(object):
             getLog().warning("specified 'pigz' as the root cache compress program but not available; using gzip")
             self.compressProgram = 'gzip'
         if self.compressProgram:
-            self.compressArgs = ['--use-compress-program', self.compressProgram]
             self.rootCacheFile = self.rootCacheFile + self.root_cache_opts['extension']
         else:
             self.compressArgs = []
@@ -186,7 +185,7 @@ class RootCache(object):
                     mockbuild.util.do(
                         ["gtar", "--one-file-system", "--exclude-caches", "--exclude-caches-under"] +
                         self.compressArgs +
-                        ["-cf", self.rootCacheFile,
+                        ["caf", self.rootCacheFile,
                          "-C", self.buildroot.make_chroot_path()] +
                         self.exclude_tar_cmds + ["."],
                         shell=False
